reo/validators.py

```diff
@@ -933,12 +1109,22 @@ def check_special_cases(self, object_name_path, template_values=None, real_value
                         'The sum of elements of percent share list for hybrid load profile should be 100.')
 
                 if real_values.get('annual_kwh') is not None:
+                    real_values['year'] = 2017
```

Comment:
this setting of the year to 2017 looks misplaced: it should only happen if using built in profile, as checked on line 1120  `if real_values.get('doe_reference_name') is not None`

Reply:
This may have been a mistake from handling previous merge conflicts.  I did a search to confirm this isn't repeated anywhere else.  This has been removed in commit 528a849c69e641c7786d67d1218011b4d57822d5